src/pages/Detail/Modal/ModalApply/ModalApply.js

```diff
@@ -0,0 +1,247 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+
+import UploadFileList from './UploadFileList/UploadFileList';
+
+const ModalApply = props => {
+  const [addFile, setAddFile] = useState([]);
+  const [disabled, setDisabled] = useState(true);
+
+  useEffect(() => {
+    const result = addFile.filter((el, i) => el.checked === true);
```

Comment:
checked 자체가 boolean입니다, i는 사용안해서 받을필요 없어보입니다.
```suggestion
    const result = addFile.filter(el => el.checked);
```